package.json

```diff
@@ -41,7 +41,7 @@
     "bottleneck": "1.5.x",
     "event-stream": "3.2.2",
     "express": "4.11.2",
-    "hiredis": "0.4.0",
+    "hiredis": "~0.4.1",
```

Comment:
(Oh, and if you decide to update the `.travis.yml` file, you should also add [these lines](https://github.com/redis/hiredis-node/blob/master/.travis.yml#L3-L4) as well to force the compiler version.)